guides/source/active_support_core_extensions.md

```diff
@@ -3467,6 +3467,7 @@ years_ago
 years_since
 prev_year (last_year)
 next_year
+on_weekday?
```

Comment:
I was going through the complete list in this file. Can you update this to have some other missing methods if any?